taclib/k8stools.py

```diff
@@ -29,45 +27,51 @@ def rerun(pod_name, new_name, tag, namespace, ipython):
     if new_name:
         pod.metadata.name = new_name
     else:
-        pod.metadata.name += "-debug"
+        pod.metadata.name += '-debug'
```

Comment:
Let's black format everything, so we don't have any formattings discussions anymore. https://github.com/psf/black

Reply:
Like this